src/core.jl

```diff
@@ -76,10 +76,20 @@ end
 # all other functions should hit one of this eventually
 # Which handles the actual testing and user prompting
 
-function _test_reference(equiv, rendermode, file::File, actual::T) where T
+function test_reference(
+    file::File,
+    actual::T,
```

Comment:
Is this type constraint ever used?

Reply:
I added this to debug an error introduced by order changes. 
I think what I really need to do here is to swap the order of `rendermode` and `equiv` so that it's consistent to

```julia
test_reference(filename::AbstractString, actual; by = nothing, render = nothing, kw...)
```

Reply:
fixed in https://github.com/Evizero/ReferenceTests.jl/pull/37/commits/8ea330db3e1383877ca9c144fd6da784cd48952d